nearcore/src/metrics.rs

```diff
@@ -15,6 +15,15 @@ pub(crate) static APPLY_CHUNK_DELAY: Lazy<HistogramVec> = Lazy::new(|| {
         .unwrap()
 });
 
+pub(crate) static DELAYED_RECEIPTS_QUEUE_LENGTH: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
I realize I've been the first to start using "Delayed receipts queue" terminology. However, thinking more about this, it won't necessarily be a queue in the future - so we might as well use something more generic like "delayed_receipt_count" here.